source/manage/server-admin-gui.rst

```diff
@@ -1,164 +1,127 @@
 .. _server-admin-gui-pmm-settings-page:
```

Comment:
Includes content duplicated by FAQ and restyled to remove use of replacements for future expansion.